src/styles/zglobals.css

```diff
@@ -0,0 +1,22 @@
+/**
+ * And why is this file now called zglobals.css? Because of https://github.com/vercel/next.js/issues/16630
+ * Some files, including _app.tsx, import both this and origin-storybook/lib/styles.css,
+ * but the order of inclusion matters because we're loading some Tailwind twice for some reason and
```

Comment:
It seems like we only get a subset of Tailwind along with the storybook components, so seems the second include is needed to fill in the gaps. I think we probably can't just bundle the whole of Tailwind in storybook for a bunch of reasons, but we may be able to configure this repo's Tailwind inclusion to avoid bringing in the declarations already brought in by storybook. That seems reasonably clean, but is probably a P3 as this seems to work for now.